testsuite/run_sets/secondary_parallelizable.yml

```diff
@@ -40,7 +40,7 @@
 - features/secondary/srv_task_status_engine.feature
 - features/secondary/srv_errata_api.feature
 
-- features/secondary/proxy_retail_pxeboot_and_mass_import.feature
+#- features/secondary/proxy_retail_pxeboot_and_mass_import.feature
```

Comment:
It would be good to have a small comment above the commented out line that states why it is commented out.